pkg/api/types.go

```diff
@@ -133,6 +133,13 @@ const (
 // OSType represents OS types of agents
 type OSType string
 
+const (
```

Comment:
Should `MasterProfile` on line 95 have `count` field?

Reply:
no, at this point it is internal api and the `MasterPoolProfile` has been converted into an `AgentPoolProfile`

Reply:
(which has count)